slackcat-example.conf

```diff
@@ -0,0 +1,7 @@
+{
+  "proxy": "http://proxy.example.com:3128",
+  "username": "slackcat",
+  "channel": "#slackcat",
+  "icon_emoji": ":family:",
```

Comment:
Honestly, I never got the icon to change, whether I was using the config file, cli, or environment variable. So I'm unsure on the format here.